VMEncryption/main/DiskUtil.py

```diff
@@ -1172,3 +1172,14 @@ def get_luks_header_size(self, device_path=None):
                 header_size = None
 
             return header_size
+            
+    def get_osmapper_name(self):
+        osmapper_name = CommonVariables.osmapper_name
+        try:
```

Comment:
Should this happen only in case of CVM? A corner case I can think of is if a customer have manually encrypted the OS volume